src/plays/simple-calculator/styles/Buttons.css

```diff
@@ -0,0 +1,54 @@
+.show-btn {
```

Comment:
Please use the unique style as a prefix to all the class names here like `.soni-calc .equal`. THis is to reduce the chances of CSS overrides.